docs/pages/build-reference/app-versions.mdx

```diff
@@ -17,7 +17,9 @@ In managed projects, we use fields `version`/`android.versionCode`/`ios.buildNum
 
 One of the most frequent causes of app store rejections is submitting a build with a duplicate version number. This happens when a developer forgets to increment the version number before running a build.
 
-EAS Build can manage automatically incrementing these versions for you if you opt into using the "remote" app version source. The default behavior is to use a "local" app version source, which means you control versions manually in their respective config files.
+EAS Build can manage automatically incrementing these versions for you if you use the "remote" app version source, which is the default behaviour. Optionally, you can choose to use a "local" app version source, which means you control versions manually in their respective config files.
```

Comment:
```suggestion
EAS Build can manage automatically incrementing these versions for you if you use the `remote` app version source, which is the default behaviour. Optionally, you can choose to use a `local` app version source, which means you control versions manually in their respective config files.
```

Remote and local are both properties in **eas.json** so let's use inline syntax highlight here (Unrelated to the changes in the PR).